js/content/common.js

```diff
@@ -2588,7 +2578,7 @@ class HorizontalScroller {
         this._controlRight = controlRightNode;
 
         this._lastScroll = 0;
-        parentNode.addEventListener("wheel", (e) => { this._scrollHandler(e); });
+        parentNode.addEventListener("wheel", e => this._scrollHandler(e));
```

Comment:
Please do mind that that these 2 lines are not equivalent. The 2nd one returns `this._scrollHandler(e)`, while the previous function doesn't return anything. That said, I can't see a problem in this case, but maybe you can thing something that may be affected by this.